src/modules/data/audioStream.c

```diff
@@ -24,13 +26,63 @@ AudioStream* lovrAudioStreamInit(AudioStream* stream, Blob* blob, size_t bufferS
   return stream;
 }
 
+AudioStream* lovrAudioStreamInitRaw(AudioStream* stream, int channelCount, int sampleRate, size_t bufferSize, size_t queueLimitInSamples) {
+  stream->bitDepth = 16;
+  stream->channelCount = channelCount;
+  stream->sampleRate = sampleRate;
+  stream->samples = SIZE_MAX;
```

Comment:
Is there a reason this is `SIZE_MAX`?  Could it be zero?

Reply:
Yeah. A realtime stream potentially never ends, so SIZE_MAX tries to say "infinite" here. Using it as a shorthand for "how long would it play NOW if I didn't queue anything more" makes sense, and then we can remove getQueueLength. Yeah, I think I like it? Slightly overloaded meaning but it works :)  I'll make the change.